content/building-blocks/collaborate-and-share-your-work/write-your-paper/forest-plot-generation-r.md

```diff
@@ -0,0 +1,180 @@
+---
+title: "Forest Plot Generation in R"
+description: "Forest Plots in R - using forest plots to gain insights on your data"
+keywords: "data, visualization, r, forest, plot"
+date: 2023-10-10
+weight: 1
+author: "Matteo Zicari"
+authorlink: "https://www.linkedin.com/in/matteozicari/"
+aliases:
+  - /visualise/data
+  - /r/forest/plots
+---
+
+## Overview
+
+`Forest plots` are a visual representation that summarises findings from various scientific studies that investigate a common research question. They find significant application in the field of `meta-analysis`, a type of statistical analysis that combines and examines results from a number of independent studies. 
+More practically, forest plots identify a statistic that is common to such set of studies and report the various instances of that statistic. This, in turn, allows to compare the different results and the significance of the overall pooled summary effect. <br/>
+<br/>
+
+Among the **benefits** of forest plots, we find:
+
+* clear and concise `visual representation` of results;
+* `effect size` and `confidence interval` comparison across different studies;
+* overall, useful tool to evaluate the consistency and strength of evidence, identify potential sources of bias, 
+  and make informed judgments about the effect of interventions or exposures.
+
+
+## Forest Plots in R
+
+One of the most popular R packages used for forest plots is [forestploter](https://cran.r-project.org/web/packages/forestploter/vignettes/forestploter-intro.html). Compared to other packages (e.g., forestplot), `forestploter` focuses entirely on forest plots, which are treated as a table. Moreover, it allows to control for graphical parameters with a theme and to have confidence intervals spread across multiple columns and divided by groups.
+
+<br/>
+
+### Generate Dataset
+
+The code snippet below shows how to `generate` a `dataset` and create a `basic layout` for a forest plot:
+
+{{% codeblock %}}
+```R
+# Load necessary packages
+library(forestploter)
+library(dplyr)
+
+# Create dataset
+data <- data.frame(
+  Study = c("Study A", "Study B", "Study C", "Study D", "Study E"),
+  Group1 = c(200, 150, 180, 250, 120),
+  Group2 = c(150, 170, 160, 240, 130),
+  smd = c(0.51, 1.27, -0.54, 0.81, 0.87),
+  CI_Lower = c(-0.25, -0.74, -1.6, 0.55, -0.1),
+  CI_Upper = c(1.25, 1.4, -0.1, 2.1, 1.3)
+)
+
```

Comment:
instead of one big codeblock, can you break this down into a few sub-tasks, so that it's easier to follow along? essentially expand out the code comments into full sentences, and put them outside the code block.